migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/sql_schema_differ_flavour/mysql.rs

```diff
@@ -51,6 +51,10 @@ impl SqlSchemaDifferFlavour for MysqlFlavour {
         None
     }
 
+    //split into enum changes
+    //family changes
+    //native type changes
```

Comment:
are these comments still relevant?

Reply:
these are just notes for the mysql work. My idea was to separate the differ logic into subcases to make it more palatable. For SQLServer that's just diffing columntypechanges vs native type changes. For Postgres lists are an additional concern. For MySQL its enums. You can remove these.